べっちさん課題/index2.html

```diff
@@ -1,28 +1,44 @@
-!DOCTYPE html>
 <html lang="ja">
 	<head>
 		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>べっちさんの課題.gitのが辛い</title>
+		<title>べっちさんの課題4</title>
 	</head>
 	<body>
 		<script>
 			"use strict";
+			const tasks = [
+				{
+					content: "机を片つける",
+					genre: "掃除",
+				},
+				{
+					content: "牛乳を買う",
+					genre: "買い物",
+				},
+				{
+					content: "散歩する",
+					genre: "運動",
+				},
+			];
 
-			function fizzBuzz() {
-				for (let i = 1; i < 100; i++) {
-					if (i % 3 === 0 && i % 5 === 0) {
-						console.log("FizzBuzz" + i);
-					} else if (i % 3 === 0) {
-						console.log("Fizz" + i);
-					} else if (i % 5 === 0) {
-						console.log("Buzz" + i);
-					}
-				}
+			function showAd() {
+				console.log("==========================");
+				console.log("現在持っているタスクの一覧");
+				console.log("==========================");
+				tasks.forEach((task, index) => {
+					console.log(
+						`${index} : [内容]${task.content}、［ジャンル]${task.genre}`
+					);
+				});
 			}
 
-			fizzBuzz();
+			showAd();
+
+			const result = prompt("タスクを入力してください");
```

Comment:
`result` と言う名前だとやや抽象的でわかりにくいです。
変数名にはなるべく具体的な名前をつける様に今から心がけましょう。

```suggestion
			const task = prompt("タスクを入力してください");
```

Reply:
ご指摘ありがとうございます！